Tribler/Core/Modules/restapi/mychannel_endpoint.py

```diff
@@ -329,10 +327,14 @@ def __init__(self, session, infohash):
     @db_session
     def render_PATCH(self, request):
         parameters = http.parse_qs(request.content.read(), 1)
-        if 'status' not in parameters:
+        if 'status' not in parameters and 'tags' not in parameters and 'title' not in parameters:
```

Comment:
The error text does not reflect the conditional (since it does not only check for status). Also, could you please document this method and give an example of a request? In particular, we should communicate the type of the `tags` parameter to the developer.

Reply:
Done;